template/next-ts-chakra-ui/.storybook/main.ts

```diff
@@ -1,29 +1,27 @@
-const path = require('path');
+import path from 'path';
+import type { StorybookConfig } from '@storybook/nextjs';
 
-module.exports = {
+const config: StorybookConfig = {
+  framework: '@storybook/nextjs',
   staticDirs: ['../public'],
   stories: [
-    '../node_modules/@portablestudios/figma2theme/lib/**/*.stories.js',
+    // TODO: Update figma2theme and uncomment this
+    // '../node_modules/@portablestudios/figma2theme/lib/**/*.stories.js',
```

Comment:
Once this is back, is there a way to make a private repository not be a dependency of this open source project?

Reply:
Nice to see you @RickMeasham 😄 `figma2theme` is already a public repository! Though I wasn't aware anyone outside of Portable was using it (or this). Either way it's an optional dependency on the generated project so easy to remove if not needed.

https://github.com/PortableStudios/figma2theme